infrastructure/channels/serialize_child.html

```diff
@@ -0,0 +1,167 @@
+<!doctype html>
+<script src="/resources/channel.sub.js"></script>
+<script src="serialize-data.js"></script>
+<script>
+
+let lastData;
+
+// Hack: these will be recreated as assert failures in the caller
```

Comment:
Why though? Can't we fail in any manner and get the same result? Does this depend on how testharness.js treats "AssertionError" differently in some place?